### DISCUSSION TITLE
How to use vda property vda5050:action.<INDEX>.when

### DISCUSSION THREAD

**Discussion by vincentknw** [openTCS/opentcs#109] [Q&A]
```

vda5050:action.<INDEX>.when
Set to the situations/triggers in which the action should be executed on a node. The property value contains a list of values, separated by |. Possible values are PASSING, ORDER_START and ORDER_END. 
```
 How should i use this property,how does it work?as far as i know, vehicle received actions in nodes,but only trigger when go through the node.

**Comment by swltr** [comment]
@vincentknw The meaning is as follows:

* `ORDER_START` means the action is only to be triggered if the node is the initial one in a route.
* `ORDER_END` means the action is only to be triggered for the last step in a route.
* `ORDER_PASSING` means the action is only to be triggered for nodes the vehicle is only passing in a route.
